WordPress/Classes/Services/ReaderPostService.h

```diff
@@ -106,6 +106,7 @@ extern NSString * const ReaderPostServiceToggleSiteFollowingState;
  @param failure block called if there is any error. `error` can be any underlying network error.
  */
 - (void)toggleLikedForPost:(ReaderPost *)post
+        fromArticleDetails:(BOOL) details
```

Comment:
Maybe we can rename `details` to `isFromArticleDetails`? Reading the method signature is clear what that means, but in the implementation inside the method `details` is not clear.

Reply:
Actually I would avoid doing this since I think it is not in line with Objective-C naming conventions. Few examples: [here](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingMethods.html#//apple_ref/doc/uid/20001282-BCIGIJJF), [here](https://google.github.io/styleguide/objcguide.html) and [here](https://stackoverflow.com/questions/15719854/naming-convention-for-local-bool-variables-in-objective-c-and-cocoa). I agree that `details` is not the most clear name, though I would not use `isFromArticleDetails` which is more Swifty. What do you think?

Reply:
Legibility > conventions IMHO. But this method is not big, so I don't think it's a big problem, you can leave it as it is.

Reply:
> Legibility > conventions

I agree, but someone coming from Obj-C might not 😊 I'm leaving it as is for now.